templates/docs/vehicle/nissan_0.yaml

```diff
@@ -0,0 +1,27 @@
+product:
+  brand: Nissan
```

Comment:
Es gibt aktuell zwei Nissan Fahrzeuge. Dieser hier und Carwings. Wenns mehr als einen gibt sollten wir immer auch das `description` Feld füllen damit wir in der Doku Unterüberschriften haben.

![Bildschirmfoto 2022-01-16 um 23 34 08](https://user-images.githubusercontent.com/152287/149680866-faf747e9-96b5-4b6c-b7b2-2fdcfac59c63.png)

Reply:
Done